l10n/de_DE/LC_MESSAGES/remoteevent.po

```diff
@@ -928,6 +932,11 @@ msgstr "Teilnahmeliste für Workshop \"%1\""
 msgid "View"
 msgstr "Ansicht"
 
+#: CRM/Remoteevent/Registration.php
+msgid "You need to be logged in to register for this event."
+msgstr ""
+"Sie müssen angemeldet sein, um sich für diese Veranstaltung anzumelden."
```

Comment:
Two times *anmelden* here, too, but this time let's rephrase that to something like *Sie benötigen ein Benutzerkonto, …*

Reply:
Changed.